ansible/roles/haproxy/defaults/main.yml

```diff
@@ -66,3 +66,6 @@ haproxy_process_cpu_map: "no"
 
 haproxy_dimensions: "{{ default_container_dimensions }}"
 keepalived_dimensions: "{{ default_container_dimensions }}"
+
+# Overwrite defaults
+haproxy_listen_neutron_server_extra: ["timeout server 15m"]
```

Comment:
Better to default this to an empty list; I think this is quite a large timeout for most users.

Reply:
I rewrote the comment. The default timeout for all services is 1m, and this is the overwrite of the default for neutron_server.